Ubuntu: Stop using StackHPC host package repos, start using StackHPC container images (Wallaby)

markgoddard opened the pull request:
* The on_demand mirror in Ark does not work if the upstream mirror pulls packages (which it does sometimes).
* A set of Ubuntu images has been built with tag `wallaby-20220819T112725`.

markgoddard committed:
Revert "Ubuntu: add host Apt repository configuration"

The Ubuntu mirrors in Ark aren't ready yet.

This reverts commit 31bc7455823f788f1becc5e063914802d68ebbb5.

markgoddard committed:
Ubuntu: stop syncing Ark Ubuntu mirrors to local Pulp

The Ubuntu mirrors in Ark aren't ready yet.

markgoddard committed:
Ubuntu: stop using Ark repo mirrors in container images

The Ubuntu mirrors in Ark aren't ready yet.

markgoddard committed:
Ubuntu: use newly built Kolla images

mnasiadka reviewed (dismissed):
I don't think there's much value in snapshotting OS repo that does proper security updates.

mnasiadka commented:
> I don't think there's much value in snapshotting OS repo that does proper security updates.

Unless we're thinking about customer deployments that only have ark.stackhpc.com whitelisted.

markgoddard commented:
> > I don't think there's much value in snapshotting OS repo that does proper security updates.
> 
> Unless we're thinking about customer deployments that only have ark.stackhpc.com whitelisted.

The value is the stability & reproducibility.

markgoddard committed:
Ubuntu: set Bifrost image tag

markgoddard committed:
CI: add hook for seed container image build

Without this, the seed tags artifact contains the overcloud images
because ~/kolla_tag hasn't been updated.

markgoddard committed:
Merge branch 'stackhpc/wallaby' into wallaby-drop-ubuntu-repos
